go.mod

```diff
@@ -16,8 +16,10 @@ require (
 	github.com/onsi/ginkgo/v2 v2.22.2
 	github.com/onsi/gomega v1.36.2
 	github.com/opencontainers/selinux v1.11.1
+	github.com/pkg/errors v0.9.1
 	github.com/safchain/ethtool v0.5.10
-	github.com/vishvananda/netlink v1.3.0
+	github.com/vishvananda/netlink v1.3.1-0.20250303224720-0e7078ed04c8
```

Comment:
this contains https://github.com/containernetworking/plugins/blob/35831f3d23956658aaa3109cbae0ce24d28137e6/go.mod#L20 that will alliviate the problem considerably

Reply:
aha, interesting -- do you have a link to the commit / patch?